src/components/elements/RouterLink.tsx

```diff
@@ -30,6 +30,7 @@ const RouterLink = forwardRef<RouterLinkProps, any>(
           gap={0.75}
           alignItems='center'
           sx={{ display: 'inline-flex', textDecoration: 'inherit' }}
+          component='span'
```

Comment:
@gigxz this is to avoid `Warning: validateDOMNesting(...): <div> cannot appear as a descendant of <p>.`

We can also fix it in the newly added alert component, but I think `RouterLink` should be allowed as a child of `<p>` in general, so I've added it here. It doesn't seem to impact the visual styling.

Reply:
Okay yeah this seems relatively low risk since it's already inside the `a` tag and it's only affecting links with icons